addons/web/static/src/views/fields/x2many/x2many_field.xml

```diff
@@ -10,7 +10,7 @@
                             <button
                                 t-if="create.type === 'create'"
                                 type="button"
-                                class="btn btn-secondary"
+                                class="btn btn-light"
```

Comment:
Actually we have to review the secondary-button color in order to mismatch the light-gray bg... sorry, you were not aware.

Could you drop this change and push--force again?
I'll merge the rest